mmv1/third_party/terraform/.teamcity/components/inputs/services_beta.kt

```diff
@@ -616,6 +616,16 @@ var ServicesListBeta = mapOf(
         "displayName" to "Securitycenter",
         "path" to "./google-beta/services/securitycenter"
     ),
+    "securitycentermanagement" to mapOf(
```

Comment:
It looks like something went wrong with the merge. I don't think you need to add this entry, do you?

Reply:
Its merged to main branch, Its showing merge conflicts happend.

Reply:
I'm not sure what you mean. Can you please resolve these conflicts so that they do not show up as part of this set of changes? I believe the merge conflicts will break our build pipeline